openedx/core/djangoapps/bookmarks/models.py

```diff
@@ -81,7 +81,7 @@ def create(cls, data):
 
             xblock_cache = XBlockCache.create({
                 'usage_key': usage_key,
-                'display_name': block.display_name,
+                'display_name': block.display_name or '',
```

Comment:
@explorerleslie Should we use `display_name_with_default` instead of ''. I think we had this conversation when we were implementing this because we are using `''` at other places too when we create breadcrumbs for a unit. Let me know what do you think.